pkg/module/go_get_fetcher_test.go

```diff
@@ -32,8 +34,34 @@ func (s *ModuleSuite) TestGoGetFetcherFetch() {
 	r.NoError(err)
 	r.True(len(zipBytes) > 0)
 
-	r.NoError(ref.Clear())
+	// close the version's zip file (which also cleans up the underlying diskref's GOPATH) and expect it to fail again
+	r.NoError(ver.Zip.Close())
 	ver, err = ref.Read()
 	r.NotNil(err)
 	r.Nil(ver)
 }
+
+func ExampleFetch() {
```

Comment:
This is a little verbose and doesn't use helper functions since it's mean't to be an example that appears in the godocs